core/src/main/scala/kafka/log/Log.scala

```diff
@@ -1128,12 +1128,12 @@ class Log(@volatile var dir: File,
    * Given a message offset, find its corresponding offset metadata in the log.
    * If the message offset is out of range, return unknown offset metadata
    */
-  def convertToOffsetMetadata(offset: Long): LogOffsetMetadata = {
+  def convertToOffsetMetadata(offset: Long): Option[LogOffsetMetadata] = {
```

Comment:
nit: can you update the method doc?

Reply:
I should pay more attention to the doc. It is updated now.